aws/v9.3.4/README.md

```diff
@@ -0,0 +1,63 @@
+# :zap: Giant Swarm Release v9.3.4 for AWS :zap:
+
+This release updates managed apps to the latest releases and includes security fixes for 3 CVEs:
+- IPv6 rogue router advertisement vulnerability [CVE-2020-13597](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13597)
+- Intel Microcode vulnerabilities [CVE-2020-0543](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-0543)
```

Comment:
It no longer includes this

Reply:
@stone-z yes. please include suggestion to remove these

Reply:
Chiara did it https://github.com/giantswarm/releases/pull/337#discussion_r453740085